otelcollector/configmaps/ama-metrics-settings-configmap.yaml

```diff
@@ -30,7 +31,10 @@ data:
     nodeexporter = ""
     windowsexporter = ""
     windowskubeproxy = ""
+    podannotations = ""
     minimalingestionprofile = true
+  default-targets-namespace-keep-list: |-
+    podannotations = ""
```

Comment:
empty --  meaning all namespaces would be included ? How would one include only a few namespaces? by specifying it here ?  For our metric keep lists - the keeplist is in addition to what we collect by default.